docs/wiki/basic-tutorials/running.md

```diff
@@ -0,0 +1,210 @@
+
+![](./running-eclipse-configurations-create.png)
+
+**7.** Run or debug your bot via the Buttons in the top row, the `Run` menu, or the shortcuts Ctrl+F11 for running and F11 for debugging.
+
+![](./running-eclipse-configurations-bar.png)
+
+## :package: Deploying and Running as a Standalone Application
+
+Running from the IDE is only recommended during development and strongly discouraged for production use. Generally, you'll want your build tool to create a convenient distribution format for you to use.
+
+### Building a Distribution with Gradle
+
+For Gradle, only two further steps are necessary for a basic application. On top of the steps described in the [Getting Started Section](/wiki/getting-started/intellij-gradle), also add the [Application Plugin](https://docs.gradle.org/current/userguide/application_plugin.html) and define your `mainClassName` as the fully qualified name of your main class. Since the `application` plugin implicitly applies the `java` plugin, you may leave that one out.
+
+Your modified `build.gradle` file should now look like this.
+```groovy
+plugins {
+    id 'application'
+}
+
+group 'com.github.yourname'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+application {
+    mainClass = 'com.github.yourname.BotMain'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.javacord:javacord:${latest-version}'
+}
+```
+
+Now you can execute the `distZip` or `distTar` task with Gradle. The task will create a distribution and package it in an archive file that will be placed in the `build/distributions` directory. Extract the content of those files on your server or whichever machine you want to run your bot on.
+
+The distribution usually only contains the directories `bin` and `lib`. From the distribution directory, run either `bin/yourbot` or `bin/yourbot.bat`, depending on whether you're running the bot on Linux or Windows.
+
+### Building a Distribution with Maven
+
+For Maven, add the [Appassembler](https://www.mojohaus.org/appassembler/appassembler-maven-plugin/usage-program.html) plugin to your `pom.xml`.
+
+```xml
+<project>
+  ...
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>appassembler-maven-plugin</artifactId>
+        <version>1.10</version>
+        <configuration>
+          <programs>
+            <program>
+              <mainClass>com.github.yourname.BotMain</mainClass>
+              <id>yourbot</id>
+            </program>
+          </programs>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</project>
+```
+
+Now you can use the `appassembler:assemble` target, for example with the invocation `mvn package appassembler:assemble`. Unlike the Gradle plugin, Appassembler will not create an archive file, so you will have to manually create one or copy the contents of the `target/appassembler` directory.
```

Comment:
As far as I remember, you should combine this with the `assembly` plugin that will then build the actual distribution archive to come to parity with Gradle `application` plugin.
It is for example essential if you bulid your bot on Windows and then want to run it on Linux, as with the assembly plugin you can add the executable flag to the packaged start scripts like the Gradle application does.